editor/src/components/canvas/canvas-utils-scene.spec.browser.ts

```diff
@@ -15,6 +15,11 @@ import { PrettierConfig } from '../../core/workers/parser-printer/prettier-utils
 import { createFakeMetadataForParseSuccess, wait } from '../../utils/test-utils'
 import { determineElementsToOperateOnForDragging } from './controls/select-mode/move-utils'
 import { BakedInStoryboardUID } from '../../core/model/scene-utils'
+import { CanvasControlsContainerID } from './controls/new-canvas-controls'
+
+// we need to set the Electron window to a larger size so document.elementsUnderPoint works correctly!
+const currentWindow = require('electron').remote.getCurrentWindow()
+currentWindow.setSize(2200, 1000)
```

Comment:
this was _very_ painful to debug :( elementsUnderPoint apparently does not work outside of an electron window's visible area… :| 

to make things worse, even though this function is synchronous, it apparently takes a moment for this to take effect. so it took me some time to realize that the resizing _is_ the solution, but it needs to happen some time before the test runs.

I feel like this is a house of cards, but I don't have a better solution than to set it at the root of the module here. this way by the time the test actually runs, the window will be resized

Reply:
hmm, now that I wrote it out "loud" , I think I probably should do something like wait for the electron resize event: https://www.electronjs.org/docs/api/browser-window#event-resize

Reply:
fixed! I pushed a much improved setup which uses `beforeAll` and waits for the resize event

Reply:
:+1: